contributor-docs/maintainers-guide.md

```diff
@@ -126,4 +128,45 @@ To run the configuration script:
         -s 1
       ```
 
-**Note**: To see optional configurable parameters, run `./configure-runners.sh -h`
\ No newline at end of file
+    **Note**: To see optional configurable parameters, run `./configure-runners.sh -h`
+
+#### CloudSQL proxy server VM
```

Comment:
Could you review/merge (or suggest updates) https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1576 before taking this? I think its worth having a separate CI/CD page and I think this would belong there